### PR TITLE
Spelling nits and style consistencies

### PR DESCRIPTION
**Pull request by SteveLasker**
Account for nits of punctionation, consistent Title Case usage and consistency with bulleted content.
This PR does not change the definition or examples of the use cases.

### CODE REVIEW

File: draft-ietf-scitt-software-use-cases.md [hunk 1]
```diff
@@ -127,11 +127,11 @@ Typically, the range of types of statements about digital products (and their de
 Taking the type and structure of all statements about digital and products into account might not be possible.
 Examples of statements may include commit signatures, build environment and parameters, software bill of materials, static and dynamic application security testing results, fuzz testing results, release approvals, deployment records, vulnerability scan results, and patch logs.
 In consequence, instead of trying to understand and describe the detailed syntax and semantics of every type of statement about digital products, the SCITT architecture focuses on ensuring statement authenticity, visibility/transparency, and intends to provide scalable accessibility.
-The following use case illustrates the scope of SCITT and elaborate on the generic problem statement above.
+The following use case illustrates the scope of SCITT and elaborates on the generic problem statement above.
 
 # Software Supply Chain Use Cases
 
-## Verification that Signing Certificate is Authorized by Supplier
+## Verification That Signing Certificate Is Authorized by Supplier
```
Comment (henkbirkholz) [inline draft-ietf-scitt-software-use-cases.md:134]:
```suggestion
## Verification that Signing Certificate Is Authorized by Supplier
```

I think we go by a "everything is lowercase except the first word and proper nouns or verbs".

**Review by henkbirkholz — APPROVED** [review]
More then spelling nits & style in `Auditing of Software Product`, but I am okay with the addition.

**Review by OR13 — APPROVED** [review]

File: draft-ietf-scitt-software-use-cases.md (hunk 1 above)
Comment (SteveLasker) [reply draft-ietf-scitt-software-use-cases.md:134]: I used: https://capitalizemytitle.com/

**Comment by SteveLasker** [comment]
> More then spelling nits & style in `Auditing of Software Product`, but I am okay with the addition.

The GitHub UX makes it look more than it is. This is just re-formatting of existing content into bullets.

**Pull request merged**